person.rb

```diff
@@ -5,14 +5,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] rentals should be in attr_accessor because it would give the ability to read and write.